package.json

```diff
@@ -13,49 +13,68 @@
     "test": "tests"
   },
   "scripts": {
-    "build": "ember build",
+    "build": "ember build --environment=production",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
-    "test": "ember test",
-    "test:all": "ember try:each"
+    "test": "npm-run-all lint test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each"
   },
   "dependencies": {
-    "ember-cli-babel": "^7.11.1",
-    "ember-cli-htmlbars": "^4.0.3"
+    "ember-cli-babel": "^7.26.10",
+    "ember-cli-htmlbars": "^5.7.2",
+    "tracked-built-ins": "^2.0.1"
   },
   "devDependencies": {
     "@babel/core": "^7.0.0",
-    "@ember/optional-features": "^1.0.0",
-    "babel-eslint": "^10.0.3",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/test-helpers": "^2.6.0",
+    "@embroider/test-setup": "^0.48.1",
+    "@glimmer/component": "^1.0.4",
+    "@glimmer/tracking": "^1.0.4",
+    "@percy/cli": "^1.0.0-beta.73",
+    "@percy/ember": "^3.0.0",
+    "babel-eslint": "^10.1.0",
     "broccoli-asset-rev": "^3.0.0",
-    "ember-cli": "~3.13.1",
+    "ember-auto-import": "^1.12.0",
+    "ember-cli": "~3.28.5",
     "ember-cli-browserstack": "^0.0.7",
-    "ember-cli-content-security-policy": "^1.1.1",
     "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-inject-live-reload": "^2.0.1",
+    "ember-cli-inject-live-reload": "^2.1.0",
+    "ember-cli-terser": "^4.0.2",
     "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-export-application-global": "^2.0.0",
-    "ember-load-initializers": "^2.1.0",
+    "ember-export-application-global": "^2.0.1",
+    "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^0.1.6",
-    "ember-percy": "^1.5.0",
-    "ember-qunit": "^4.5.1",
-    "ember-resolver": "^5.3.0",
-    "ember-source": "~3.13.0",
-    "ember-source-channel-url": "^2.0.1",
-    "ember-template-lint": "^1.1.0",
-    "ember-try": "^1.2.1",
-    "eslint": "^6.5.1",
-    "eslint-config-prettier": "^6.4.0",
-    "eslint-plugin-ember": "^7.1.0",
-    "eslint-plugin-node": "^10.0.0",
-    "eslint-plugin-prettier": "^3.0.1",
+    "ember-page-title": "^6.2.2",
```

Comment:
I don't think we `ember-page-title`

Reply:
Yeah, this is a new one that gets added via the `ember-cli-update`, but not relevant for addons -- will remove